jupyter-covid19-restrict-download/Dockerfile

```diff
@@ -0,0 +1,90 @@
+ARG ROOT_CONTAINER=quay.io/cdis/ubuntu:focal
```

Comment:
- can you rename `jupyter-covid19-restrict-download` to `jupyter-restricted-download` as mentioned in the ticket comments (since it's not specific to PRC but just a new base image),
- and add [this workflow](https://github.com/uc-cdis/containers/blob/copy-image-jupyter-covid19-restrict-download/.github/workflows/build_jupyter_covid19_restricted_download.yml) that i had shared in the ticket as well (the file paths will need to be renamed there too)?

Reply:
Okay. You should be able to see the changes in the upcoming commit

Reply:
I have changed the workflow file but the file name is still build_jupyter_covid19_restricted_download.yml
as I didn't know if I could change the file or not